modules/bound-sa-tokens-configuring.adoc

```diff
@@ -67,13 +67,32 @@ If the output shows a message similar to one of the following messages, the upda
 ** `3 nodes are at revision 11; 0 nodes have achieved new revision 12`
 ** `2 nodes are at revision 11; 1 nodes are at revision 12`
 
-... Manually restart all pods in the cluster:
+.. Optional: Force the holder to request a new bound token either by performing a rolling node restart or by manually restarting all pods in the cluster.
++
+// TODO: When would a user NOT need to do this?
+
+*** Perform a rolling node restart:
++
+[WARNING]
+====
+If you have workloads running on your cluster, performing a rolling node restart can cause a service interruption. TODO: What to say about how to "be careful" to minimize disruption?
+
+// TODO: Are the warnings different enough between node restarts vs pod deletions to keep them separate, or should we combine them?
+====
++
+Restart nodes sequentially. Wait for the node to become fully available before restarting the next node. See _Rebooting a node gracefully_ for instructions on how to drain, restart, and mark a node as schedulable again.
++
+// TODO: Want the piece about making sure that each node becomes fully available (Ready status) before restarting the next one?
```

Comment:
@stlaz @mfojtik I added something in the above sentence about making sure to wait til the node is fully available before starting the next node restart. Might be general knowledge, but figured it was still probably worth mentioning. Let me know if that wording looks okay or if you'd adjust it somehow

Reply:
@stlaz Thoughts on this question?

Reply:
I would just point the customers to follow the section _Rebooting a node gracefully_ right after `Restart nodes sequentially.`